### ISSUE TITLE
Support for Docusaurus 3.3.2

### ISSUE THREAD

**Issue by timolagus** [typesense/docusaurus-theme-search-typesense#45]
## Description

Add support for [Docusaurus 3.3.2](https://docusaurus.io/changelog/3.3.2).

## Steps to reproduce

Update to the following dependency versions in `package.json`:

```
@docusaurus/core: 3.3.2
@docusaurus/module-type-aliases: 3.3.2
@docusaurus/preset-classic: 3.3.2
@docusaurus/theme-live-codeblock: ^3.3.2
@docusaurus/theme-mermaid: ^3.3.2
@docusaurus/types: 3.3.2
```

## Expected Behavior

`npm install` completes without issue.

## Actual Behavior

`npm install` fails with:

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR!
npm ERR! While resolving: docusaurus-theme-search-typesense@0.18.0
npm ERR! Found: @docusaurus/core@3.3.2
npm ERR! node_modules/@docusaurus/core
npm ERR!   @docusaurus/core@"3.3.2" from the root project
npm ERR!   @docusaurus/core@"3.3.2" from @docusaurus/plugin-debug@3.3.2
npm ERR!   node_modules/@docusaurus/plugin-debug
npm ERR!     @docusaurus/plugin-debug@"3.3.2" from @docusaurus/preset-classic@3.3.2
npm ERR!     node_modules/@docusaurus/preset-classic
npm ERR!       @docusaurus/preset-classic@"3.3.2" from the root project
npm ERR!   24 more (@docusaurus/plugin-google-analytics, ...)
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @docusaurus/core@"3.2.1" from docusaurus-theme-search-typesense@0.18.0
npm ERR! node_modules/docusaurus-theme-search-typesense
npm ERR!   docusaurus-theme-search-typesense@"^0.18.0" from the root project
npm ERR!
npm ERR! Conflicting peer dependency: @docusaurus/core@3.2.1
npm ERR! node_modules/@docusaurus/core
npm ERR!   peer @docusaurus/core@"3.2.1" from docusaurus-theme-search-typesense@0.18.0
npm ERR!   node_modules/docusaurus-theme-search-typesense
npm ERR!     docusaurus-theme-search-typesense@"^0.18.0" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

## Metadata

**Typesense Version**: 0.25.1

**OS**: Ubuntu 22.04.3 LTS

**Comment by jasonbosco** [comment]
I missed the notification for your original issue. Will push out an update shortly. 

re: your side note, you'd typically want to first fork the repo into your GitHub profile, then make changes, commit to your repo and then send a pull request from your repo to the source repo. So you don't need permissions on the source repo to send PRs.

**Comment by jasonbosco** [comment]
Just published 0.19.0-0. Could you give it a shot and let me know how it goes?

**Comment by jasonbosco** [comment]
Thank you for confirming. Just published `0.19.0`.
